### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Holger-Harmsen** [NumEconCopenhagen/projects-2020-emil-kirstine-thomas#4]
Cool parts
I like that you really show your data cleaning, transforming and reading skills. You are true DataFrame-heroes. 
- Nice transposing the data
- Nice removing the top horizontal index
- Feel like it being risky to work with totals, when the subgroups do not add up to this as you leave some out - would be nice with a comment clarifying this. Very small detail though.
- Some sort of story with the project would be cool; clarifying what you’re seeking to test or analyze, drawing up the framework for your project. Just providing the opportunity to draw some conclusions based on the data you work with.

Unclear parts
- Cool that you calculate the yearly growth-rates. I am, however, a bit sceptic of descriptive inference on growth-rates, other than maybe the average rates (which you do comment on). Is standard deviation in growth rates relevant? They all clearly have very high highs and low lows. I feel this makes interpreting other parts than average growth difficult.
- Why is it relevant to show that your data is of float-type?
- Would like some more visualization, for an example graphs of the real levels of im/exports over the period. I don’t feel Hjarn the Great would call it kosher to show yearly growth as a graph rather than as a bar chart

In conclusion:
The best part of the project is your rigorous data manipulation with pandas. 
The hardest part to understand was the point of what you did. What did you want to show? What could you conclude?
It would be nice with more documentation of what could actually be drawn qualitatively from the data; maybe using descriptive inference on the levels instead of the growth-rates, and plotting some more. 
An improvement would be more (and prettier) graphs.
An extension could be a greater scope of the project. Maybe setting up a story you want to tell with the project (even though it might be BS), I’d assume it would be more fun to do the project if you have some sort of intention with it.
Great project though, wish I had your DF-manipulation skills!
